packages/wonder-blocks-button/components/button-core.js

```diff
@@ -276,16 +276,16 @@ const _generateStyles = (color, kind, light, iconWidth) => {
             default: {
                 background: "none",
                 color: light ? white : color,
-                paddingLeft: 4,
-                paddingRight: 4,
+                paddingLeft: 0,
+                paddingRight: 0,
             },
             focus: {
                 ":after": {
                     content: "''",
                     position: "absolute",
                     height: 2,
-                    width: `calc(100% - 8px - ${iconWidth}px)`,
-                    right: 4,
+                    width: `calc(100% - ${iconWidth}px)`,
```

Comment:
This could probably be simplified further but would require some restructuring so we'll leave it as is for now.